pkgs/development/python-modules/pytest-md-report/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, pytablewriter
+, pytest
+, tcolorpy
+, typepy
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-md-report";
+  version = "0.3.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    pname = "pytest-md-report";
+    inherit version;
+    hash = "sha256-ra88WXG6+xHSjOcy6tdYjvpKpNlvu6lq+sZckLadAlU=";
+  };
+
+  propagatedBuildInputs = [
+    pytablewriter
+    pytest
```

Sometimes it's necessary that we are able to override pytest. Thus it should go to `buildInputs`

```suggestion

  buildInputs = [
    pytest
  ];

  propagatedBuildInputs = [
    pytablewriter
```

---

ah, I didn't consider that; fixed in the latest push, thanks